doc/nrf/ug_matter_configuring_env.rst

```diff
@@ -172,3 +172,72 @@ To configure and use Thread Border Router and Matter controller on the same devi
    * Prepare the environment for testing by completing the steps listed in "Using CHIP Tool for Matter device testing".
 
 #. Depending on which Matter sample you programmed onto the development kit, go to this sample's documentation page and complete the steps from the Testing section.
+
+.. _ug_matter_configuring_pc_chip_tool_wifi:
+
+Matter over Wi-Fi: Configuring CHIP Tool for Linux or macOS
+***********************************************************
+
+In this setup, the Matter controller is installed on PC that is running either Linux or macOS, and a dedicated Wi-Fi Access Point is used.
+This is the recommended setup.
+
+Requirements
+------------
+
+To use this setup, you need the following hardware:
+
+* 1x PC with Ubuntu (20.04 or newer)
+* 1x Wi-Fi Access Point supporting IPv6 (without the IPv6 Router Advertisement Guard enabled on the router)
+* 1x compatible Nordic Semiconductor's DK - for the Matter accessory device (compatible and programmed with one of :ref:`matter_samples`)
+
+Configuring the environment
+---------------------------
+
+To configure and use CHIP Tool for Linux or macOS on separate devices, complete the following steps:
```

Comment:
What is on separate devices - CHIP Tool and ... ? I think it could be some leftover after Thread Border Router and controller running on separate devices?